node-js-basics-101/node_modules/uws/uws.js

```diff
@@ -0,0 +1,563 @@
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->